intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,19 +129,29 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify two main things: their **state** and their **behavior**.
+
+In the context of Object-Oriented Programming, **state** is used to refer to all the variables packaged within a class and **behavior** is used to refer to all the functions packaged within a class.
+
+### Attributes and Methods
+
+There are two more words you will often see in the context of OOP: **attribute** and **method**.
+
+For whatever reason, engineers long-ago decided we needed new words to describe variables and functions that make up a class's state and behavior. 
```

Comment:
I almost feel like we need to have a disclaimer here about how engineers like to introduce new terminology so that they have something to call each other out for when we invariably use the wrong term. I love precision as much as the next programmer, and when we're all on the same page, context-specific vocab can be helpful. So maybe I'm ok with the level of whimsy here, but I'd almost like to see it ratcheted up!